tests/test_adjoint.py

```diff
@@ -155,7 +165,7 @@ def test_adjoint_J(self, mkey, shape, kernel, space_order, time_order, setup_fun
         dot test:
         < Jx, y> = <x ,J^T y>
         """
-        tn = 500.  # Final time
+        tn = 1000. if mkey[-3:] == "-fs" else 500.  # Final time (twice the time for fs)
```

Comment:
Why?

Reply:
the data in the linearized shot record is close to the end of the register time with very low values and that makes the values of the dot product be very low, that's why I increased the time, although it can be a lower value for example 750.